setup.py

```diff
@@ -19,20 +19,20 @@
 try:
     from Cython.Build import cythonize
 except ImportError:
-    os.system('pip install Cython')
+    os.system('pip3 install Cython')
     from Cython.Build import cythonize
 
 # Cykhash needs to be installed before running setup
 try:
     import cykhash
 except ImportError:
-    os.system('pip install https://github.com/HTenkanen/cykhash/archive/master.zip')
+    os.system('pip3 install https://github.com/HTenkanen/cykhash/archive/master.zip')
```

Comment:
This can actually be nowadays installed from [PyPi](https://pypi.org/project/cykhash/) directly.